File: ApiRestCuenta/Entidad/Cancion.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Threading.Tasks;

namespace ApiRestCuenta.Entidad
{
    public class Cancion
    {
        
        public Cancion(int id, string nombre,string genero,string duracion, int albumId){
            this.id=id;
            this.nombre=nombre;
            this.genero=genero;
            this.duracion=duracion;
            this.Album_id = albumId;
        }

         public Cancion(){
        }

        [Key]
        public int id { get; set; }
        public string nombre { get; set; }
        public string genero { get; set; }
        public string duracion { get; set; }
        public int ListaDeReproduccionId { get; set; }
        public int Album_id { get; set; }

    }
}

File: ClienteStreaming/app/src/main/java/com/example/clientestreaming/Cancion.java
package com.example.clientestreaming;

public class Cancion {

    private int id;
    private String nombre;
    private String genero;
    private String duracion;
    private String audio;
    private int ListaDeReproduccionId;
    private int idAlbum;

    public Cancion(int id, String nombre, String genero, String duracion, String audio, int ListaDeReproduccionId, int idAlbum) {
        this.id = id;
        this.nombre = nombre;
        this.genero = genero;
        this.duracion = duracion;
        this.audio = audio;
        this.ListaDeReproduccionId = ListaDeReproduccionId;
        this.idAlbum = idAlbum;
    }

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public String getNombre() {
        return nombre;
    }

    public void setNombre(String nombre) {
        this.nombre = nombre;
    }

    public String getGenero() {
        return genero;
    }

    public void setGenero(String genero) {
        this.genero = genero;
    }

    public String getDuracion() {
        return duracion;
    }

    public void setDuracion(String duracion) {
        this.duracion = duracion;
    }

    public String getAudio() {
        return audio;
    }

    public void setAudio(String audio) {
        this.audio = audio;
    }

    public int getListaDeReproduccion() {
        return ListaDeReproduccionId;
    }

    public void setListaDeReproduccion(int listaDeReproduccion) {
        this.ListaDeReproduccionId = listaDeReproduccion;
    }

    public int getAlbum_id() {
        return idAlbum;
    }

    public void setAlbum_id(int album_id) {
        idAlbum = album_id;
    }
}
